dist/icon/background/ds4/background.css

```diff
@@ -1,6 +1,5 @@
 span.icon {
   background-repeat: no-repeat;
-  background-size: contain;
```

Comment:
@seangates Just checking that this removal is intended...

Reply:
Oooh, yeah. That's not intended. I'll take a look.

Reply:
Okay, so this is correct. Note the `background-size` a few rules down.

https://github.com/eBay/skin/blob/890af66cfea753a018b9be741b0fcd359300b8f2/dist/breadcrumb/ds4/breadcrumb.css#L19